index.html

```diff
@@ -0,0 +1,318 @@
+                            </div>
+                        </ul>
+                    </div>
+                </section>
+                <!-- skills -->
+                <section class="skill-card">
+                    <!-- icon -->
+                    <div class="skill-card-icon">
+                        <div class="skill-card-icon-container"><img class="skill-card-icon-img" src="img/skill/skills-img.svg" alt=""></div>
+                    </div>
+                    <!-- title and tags -->
+                    <div  class="skill-card-content">
+                        <h3 class="skill-card-title">Skills</h3>
+                        <ul class="skill-card-rows">
+                            <div class="skill-card-tech">
+                                <li class="skill-card-tech-tag codekit"><p class="skill-card-tech-tag-text">Codekit</p></li>
+                                <li class="skill-card-tech-tag github"><p class="skill-card-tech-tag-text">GitHub</p></li>
+                                <li class="skill-card-tech-tag codepen"><p class="skill-card-tech-tag-text">Codepen</p></li>
+                                <li class="skill-card-tech-tag gitlab"><p class="skill-card-tech-tag-text">Gitlab</p></li>
+                            </div>
+                            <div class="skill-card-tech">
+                                <li class="skill-card-tech-tag terminal"><p class="skill-card-tech-tag-text">Terminal</p></li>
+                            </div>
+                        </ul>
+                    </div>
+                </section>
+            </div>
+        </section>
+        
+        <!-- Form Section -->
+        <section id="form">
+            <!-- text -->
+            <div id="form-text">
+                <p id="form-text-item">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</p>
+            </div>
+            <!-- form -->
+            <form action="https://formspree.io/f/myyawpqy" method="POST" id="form-content">
+                <div class="form-row" id="first-form-row">
+                    <input type="text" required name="first-name" placeholder="First Name" class="form-row-column">
+                    <input type="text" required name="last-name" placeholder="Last Name" class="form-row-column">
+                </div>
+                <input type="text" required name="full name" placeholder="Full Name" class="form-row" id="ignoreindesktop">
+                <input type="email" required name="email" placeholder="Email Address" id="" class="form-row">
+                <textarea name="message" id="" cols="30" rows="10" placeholder="Typing..." class="form-row textareas"></textarea>
+                <div class="button-div">
+                    <button type="submit" class="form-submit-button"><p class="form-submit-button-text">Get in touch </p></button>
+                </div>
+            </form>       
```

Comment:
- [ ] Please, I suggest that you maintain a very pixel-perfect design like you'll be doing in your very soon coming job. 😉 

Kindly add padding before these placeholders. 😀 

- Your design
<img width="500" alt="wrong" src="https://user-images.githubusercontent.com/41163571/220450933-e14a4a74-ac43-4e87-b32c-d124f8b540b0.png">


- Figma design
<img width="155" alt="right" src="https://user-images.githubusercontent.com/41163571/220451053-7ded5c4a-e4f8-4ca0-93ca-b4589799d1d1.png">